public/javascripts/Gallery/src/displays/SeverityDisplay.js

```diff
@@ -20,20 +20,20 @@ function SeverityDisplay(container, severity, labelType, isModal=false) {
 
     let circles = [];
     function _init() {
-        // Set the different classes and ids depending on whether the severity display is in a Modal or in a card.
-        let severityCircleClass = isModal ? 'modal-severity-circle' : 'severity-circle';
-        let selectedCircleID = /*isModal ? 'modal-current-severity' : */'current-severity';
+        // Set the different classes and ids depending on whether the severity display is in expanded view or in a card.
+        let severityCircleClass = isExpandedView ? 'expanded-view-severity-circle' : 'severity-circle';
+        let selectedCircleID = 'current-severity';
```

Comment:
I was suggesting that we fully remove the variable! No need to save the variable and then use it just once.